relay.d.ts

```diff
@@ -0,0 +1,43 @@
+export declare const re: {
+  BINARY_BROADCAST: Uint8Array,
+  RELAY_QUERY: Uint8Array,
+  TEXT_BROADCAST: string,
+  BINARY: number,
+  TEXT: number,
+  OP: {
+    ANNOUNCE: number,
+    ANNOUNCE_REPLY: number,
+  },
+  USERS: {},
+  UInt8UserIdToBase64: (userId: Uint8Array) => string,
+  Base64ToUInt8UserID: (userId: string) => Uint8Array,
+  ArrayToStr: (x: array) => string,
+  StrToArray: (x: string) => array,
+  UserIdHex: (userId: Uint8Array) => string,
+};
+
+export declare const ArrayEq: (a: array, b: array) => boolean;
+export declare const UInt8UserIdToBase64: (userId: Uint8Array) => string;
+export declare const Base64ToUInt8UserID: (userId: string) => Uint8Array;
+
+export declare class Relay {
+  public ws: WebSocket;
+  public onClientDisconnect: ((userId: Uint8Array) => void) | null;
+  public onSubscription: ((userId: Uint8Array) => Relay) | null;
+  public BinaryMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
+  public TextMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
```

Comment:
Shouldn't `message` here be a string?

Reply:
Never mind it looks like the code is converting the string to object before passing, so it should just be a generic `Object` type.  Line above, likewise should always be `Uint8Array` type from what I can see.

Reply:
Are you sure about type on `sender`?